index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>apple-clone</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+</head>
+<body>
+    <main>
+        <div class = "main-image">
+            <nav>
+                <ul>
+                    <li>
+                        <button class="apple-icon">
+                            <i class="fa fa-apple" aria-hidden="true"></i>
+                        </button>
+                    </li>
+                    <li>
+                        <button>Store</button>
+                    </li>
+                    <li>
+                        <button>Mac</button>
+                    </li>
+                    <li>
+                        <button>iPod</button>
+                    </li>
+                    <li>
+                        <button>iPhone</button>
+                    </li>
+                    <li>
+                        <button>iPad</button>
+                    </li>
+                    <li>
+                        <button>iTunes</button>
+                    </li>
+                    <li>
+                        <button>Support</button>
+                    </li>
+                    <li class="right button">
+                        <i class="fa fa-search custom-user"  aria-hidden="true"></i><input type="search" name="search">
+                    </li>
+                </ul>
+            </nav>
```

Comment:
- [ ]  Read this [article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) and this [question](https://stackoverflow.com/questions/21715162/when-should-i-use-a-button-button-or-a-link-a-in-html) about the use of buttons. You shouldn't use them the way you are using them right now (please apply the above advice to change them).

- [ ]  The `nav` tag shouldn't be inside the `main` tag. It should be before.